packages/dds/tree/src/test/feature-libraries/chunked-forest/codec/uncompressedEncode.spec.ts

```diff
@@ -16,14 +16,18 @@ import {
 import { ajvValidator } from "../../../codec/index.js";
 import { testTrees } from "../../../cursorTestSuite.js";
 import { jsonableTreesFromFieldCursor } from "../fieldCursorTestUtilities.js";
+import { testIdCompressor } from "../../../utils.js";
 
 describe("uncompressedEncode", () => {
 	// TODO: test non size 1 batches
 	describe("test trees", () => {
 		for (const [name, jsonable] of testTrees) {
 			it(name, () => {
 				const input = cursorForJsonableTreeField([jsonable]);
-				const context = { encodeType: TreeCompressionStrategy.Uncompressed };
+				const context = {
```

Comment:
@CraigMacomber might disagree, but should we add an identifier field to the testTrees imported above?

Reply:
The above testTrees are from cursorTestSuite. Those trees are not associated with an idCompressor, and cursors do nothing based of field kinds, so that seems like it would be the wrong place to add coverage for identifiers.

The suite of testTrees from testTrees.ts would make more sense for such use, and does appear to be updated to add more cases. Perhaps they should be used instead.